docs/distribution_methods.md

```diff
@@ -18,15 +18,75 @@ Employee apps are distributed through [MDM inTune by the OCIO Device Management
 
 [Contact us](contact.md) at the start of your project so we can discuss the process for distributing an internal app. It requires setup from both the Developer Experience and OCIO Device Management teams.
 
-Apps used by both employees and contractors also need extra setup in Apple's App Store Connect. The [Developer Experience Team](contact.md) will assist in this setup.
+### Apple Custom App 
+
+Use this distribution method for internal apps used only by employees. These apps are either made specifically for the Province of BC or are a SaaS solution. 
+
+Custom apps go through Apple's app review process. The same app review guidelines for App Store Apps apply to Custom Apps.
+
+Custom Apps are licensed from [Apple Business Manager (ABM)](https://support.apple.com/en-ca/guide/apple-business-manager/welcome/web) and assigned to the OCIO Device Management team's Intune Tenancy. They will distribute the app through MDM-Intune. 
+
+#### Product Owner responsibilities
+
+The Product Owner will work with the relevant teams to decide whether to use the OCIO’s ABM or the Ministry’s own ABM.
+
+If the Ministry decides to use its own ABM, it would need to create and manage it. As part of the setup it would add the OCIO’s MDM-Intune Organization ID as a location.
+
+The Product Owner will provide the ABM ID to the developer.
+
+
+#### Developer responsibilities
+
+The developer handles developing the app and uploading it to [App Store Connect](https://appstoreconnect.apple.com). They will make the Custom App available to the appropriate ABM. Refer to Apple's [Distributing Custom Apps](https://developer.apple.com/custom-apps/) documentation for  instructions.
+
+
+#### OCIO Device Management team responsibilities
+
+The OCIO Device Management Team will distribute the app to employee devices. They will work with the Product Owner to schedule the release.
+
+
+### Apple Enterprise App
+
+Use this distribution method for internal apps used only by employees. These apps are specifically made for the Province and are not SaaS solutions. This distribution method is for apps that can't go through Apple's app review process.
```

Comment:
It sounds like we want to discourage new applications from using this, so perhaps let's add something here to that effect? (if I'm remembering our conversations correctly?)